app/api/discount_codes.py

```diff
@@ -70,6 +73,63 @@ def validate_quantity(self, data, original_data):
                          type_='event')
 
 
+class DiscountCodeSchemaEvent(Schema):
```

Comment:
@poush I initially tried that but I was not able to change relationships for the other after inherit. And the name `events` <-> `event` are done on purpose. I wouldn't be able to change it if I were to inherit the whole main schema?